### PR TITLE
Standardize README and add License

### PR DESCRIPTION
**Pull request by RichardLitt**
See ipfs/community#124

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,8 +84,18 @@ The specs contained in this repository are:
...
+
+Feel free to join in. All welcome. Open an [issue](https://github.com/ipfs/specs/issues)!
+
+This repository falls under the IPFS [Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md).
+
+[![](https://cdn.rawgit.com/jbenet/contribute-ipfs-gif/master/img/contribute.gif)](https://github.com/ipfs/community/blob/master/contributing.md)
+
+## License
+
+[CC-BY-SA](LICENSE)
```
Comment (JesseWeinstein) [inline README.md]: Should you mention the version and unported-ness here, also?

File: README.md [hunk 2]
```diff
@@ -25,7 +27,7 @@ Nothing in this spec repository is `permanent` yet. The most important pieces of
 
 Note that, as in many IPFS repositories, most of the work is happening in [the issues](https://github.com/ipfs/specs/issues/) or in [active pull requests](https://github.com/ipfs/specs/pulls/). Go take a look!
```
Comment (JesseWeinstein) [inline README.md]: Missing `Table of Contents` section.

Comment (JesseWeinstein) [inline README.md]: `Install` and `Usage` sections are technically required, but that's likely a bug in the requirements, as they don't actually seem relevant here.

Comment (JesseWeinstein) [inline README.md]: `Install` and `Usage` sections are technically required, but that's likely a bug in the requirements, as they don't actually seem relevant here.

**Comment by JesseWeinstein** [comment]
Various quibbles and suggestions.

File: README.md (hunk 1 above)
Comment (RichardLitt) [inline README.md]: Not sure what you mean by unported-ness. Can you clarify for me? :)

File: README.md (hunk 2 above)
Comment (RichardLitt) [inline README.md]: Added. Those are not needed, document repo.

File: README.md (hunk 1 above)
Comment (JesseWeinstein) [inline README.md]: Regarding version -- the most current version of CC-BY-SA is 4.0, and it is different than the previous versions. The version should probably be mentioned. Regarding unported-ness, in the 3.0 version, there were "ports" to different countries legal systems (e.g. 

See https://creativecommons.org/faq/#should-i-choose-an-international-license-or-a-ported-license and https://wiki.creativecommons.org/wiki/License_Versions for more detail.

Comment (RichardLitt) [inline README.md]: Thanks. Let's just go with 4.0. Good call.

**Comment by RichardLitt** [comment]
This needs @jbenet sign-off for the license. Might be a good idea to split this PR into two, and just have the License stuff in a separate PR. @JesseWeinstein You think the other Readme edits are fine?

**Comment by JesseWeinstein** [comment]
Yep, the other ones look good to me.

**Comment by RichardLitt** [comment]
Ok. I opened https://github.com/ipfs/specs/issues/137 to deal with that; one last review to make sure this looks ok, and then we can merge it.

**Comment by RichardLitt** [comment]
Thanks @JesseWeinstein!

**Pull request merged**